packages/sdk/package.json

```diff
@@ -8,35 +8,50 @@
     "repository": {
         "type": "git",
         "url": "https://github.com/Concordium/concordium-node-sdk-js",
-        "directory": "packages/common"
+        "directory": "packages/sdk"
     },
     "sideEffects": false,
```

Comment:
Since we are enabling polyfills for the user just by importing the package, this is not without sideeffects anymore. Not sure if the bundlers for react native respect this field

Reply:
I dont know, but we're exporting a UMD, which cannot be tree-shaken anyway. I can mark the react-native UMD as having side effects though.